spec/features/account_creation/completions_cancel_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.feature 'asks users if they want to exit to partner agency', allowed_extra_analytics: [:*] do
```

Comment:
1. `allowed_extra_analytics` is intended for legacy specs which need to be updated to ensure all analytics method arguments are included in the method signature. We should remove it and fix any issues that come up.
2. I might include something in the test description about this being related to cancelling at the completions screen

```suggestion
RSpec.feature 'canceling at the completions screen' do
```

Reply:
> 1. `allowed_extra_analytics` is intended for legacy specs which need to be updated to ensure all analytics method arguments are included in the method signature. We should remove it and fix any issues that come up.
> 2. I might include something in the test description about this being related to cancelling at the completions screen

It appears that any file that uses `visit_idp_from_sp_with_ial1(:oidc)` requires that we pass through the `allowed_extra_analytics: [*]` option. I think what is happening is that when analytics events are called throughout the testing process, it requires params through every step of the feature test.  See below error--this happens when we go from the SP to the home page

```
     Failure/Error:
                 raise UndocumentedParams, <<~ERROR
                   event :#{method_name} called with undocumented params #{extra_keywords.inspect}
                   (if these params are for specs only, use :allowed_extra_analytics metadata)
                 ERROR

     FakeAnalytics::UndocumentedParams:
       event :openid_connect_request_authorization called with undocumented params [:success, :errors, :prompt, :allow_prompt_login, :code_challenge_present, :service_provider_pkce, :referer]
       (if these params are for specs only, use :allowed_extra_analytics metadata)
```
Based on this, I am not sure if we can use the helper without passing through `allowed_extra_analytics: [*]`. Thoughts?

Reply:
Let's just leave the `allowed_extra_analytics` for now